config/processCcd.py

```diff
@@ -56,12 +56,15 @@
 config.charImage.load(os.path.join(configDir, "cmodel.py"))
 config.charImage.measurement.load(os.path.join(configDir, "apertures.py"))
 config.charImage.measurement.load(os.path.join(configDir, "kron.py"))
+config.charImage.measurement.load(os.path.join(configDir, "hsm.py"))
+if "ext_shapeHSM_HsmShapeRegauss" in config.charImage.measurement.plugins:
+    # no deblending has been done
+    config.charImage.measurement.plugins["ext_shapeHSM_HsmShapeRegauss"].deblendNChild = ""
+    config.charImage.measurement.slots.shape = "base_SdssShape"
```

Comment:
If we think HSM is superior, why should we use the SDSS shape here?  I think that if that's what we're doing on HSC, it's by accident.